pjmedia/include/pjmedia/config.h

```diff
@@ -1619,6 +1619,29 @@
 #endif
 
 
+/**
+ * Specify the number of keep-alive needed to be sent after the stream is
+ * created.
+ *
+ * Setting this to 0 will disable it.
+ *
+ * Default : 0
+ */
+#ifndef PJMEDIA_STREAM_START_KA_CNT
+#   define PJMEDIA_STREAM_START_KA_CNT	0
```

Comment:
perhaps setting the default to 1-3 is good (e.g: NAT hole punch is good for video to avoid missing keyframe)?

Reply:
done